README.md

```diff
@@ -156,10 +202,72 @@ headers:
 "Content-Type": "application/json"`, 
 "Accept": "application/json"
 ```
-Required params: "id", "profession"
+Required params: "profession"
 Example body:
 ```
 {
     "profession": "doctor"
 }
 ```
+## Roadmap
+The team utilized a [github project board](https://github.com/orgs/Head-To-Toe/projects/1) to organize project issues and coordinate workflow across 5 members and 2 repositories over a 3 week stretch.
+See the [open issues](https://github.com/Head-To-Toe/headToToe-be/issues) for a list of proposed features (and known issues).
+
+<!-- CONTRIBUTING -->
+## Contributing
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**. We hope that users provide input to improve their experience and that of others.
+
+###### Versions
+- Ruby 3.0.0
+- Rails 6.1.4
+  
+###### Making a Contribution
+1. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+2. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+3. Push to the Branch (`git push origin feature/AmazingFeature`)
+4. Open a Pull Request from your fork to this main repo
+
+<!-- CONTRIBUTORS -->
+### Contributors
+##### Backend Team
+Austin Andrade [![Github][github-shield]][austin-github-url] [![LinkedIn][linkedin-shield]][austin-linkedin-url] [![Email][email-shield]][austin-email-url] <br>
+Aidan Mcguire [![Github][github-shield]][aidan-github-url] [![LinkedIn][linkedin-shield]][aidan-linkedin-url] [![Email][email-shield]][aidan-email-url] <br>
```

Comment:
Capitalize the 'G' in McGuire for me please 🤠

Reply:
> Capitalize the 'G' in McGuire for me please 🤠

Noted!